src/garage/tf/algos/ddpg.py

```diff
@@ -4,7 +4,7 @@
 from dowel import logger, tabular
 import numpy as np
 import tensorflow as tf
-import tensorflow.contrib as tc
+from tensorflow import contrib as tc
```

Comment:
does `# pylint: disable=import-error` work instead? i would rather it be there with a comment to explain the flaw for the next person then just picking a magic import string which works.